velox/common/caching/tests/AsyncDataCacheTest.cpp

```diff
@@ -1129,21 +1128,25 @@ DEBUG_ONLY_TEST_F(AsyncDataCacheTest, ttl) {
 
   stt.setCurrentTestTimeSec(loadTime1);
   loadNFiles(filenames_.size() * 2 / 3, offsets);
+  while (cache_->ssdCache()->writeInProgress()) {
```

Comment:
Maybe put this into a test utility: waitForSSDWriteToFinish()? thanks!

Reply:
Thanks, @xiaoxmeng , for the review! Revised according to above.